src/components/Forms/Artist/index.tsx

```diff
@@ -14,9 +20,23 @@ export default function ArtistForm() {
         handleDeletePhoto,
         artistMutationIsSuccess,
         editArtistMutationIsSuccess,
-        error
+        error,
+        reset,
+        getValues
     } = useArtistForm()
 
+    useEffect(() => {
```

Comment:
This code block is not formatted correctly. Please run prettier on this file.

Reply:
Dude, look at the indentation - it's all fucked up. 

ex: ```ts
   })
        }
     ```

Reply:
Indentation is not uneven on lines 36 & 37. I don't see anywhere where it's uneven or inconsistent with the rest of the codebase.

Reply:
Looking good now, looks like you fixed it.